### DISCUSSION TITLE
How to use SymbolManager in 11.0.0 version ?

### DISCUSSION THREAD

**Discussion by Nanamare** [maplibre/maplibre-native#2391] [Q&A] [answered]
When upgrading to version 11.0.0, 
MapView and MapboxMap were renamed to MaplibreView and MapLiberMap,
So How can I use the `org.maplibre.gl:android-plugin-annotation-v9` dependency?

**Comment by louwers** [comment, marked as answer]
Not yet, it's in the works.

https://github.com/maplibre/maplibre-plugins-android/pull/52
https://github.com/maplibre/maplibre-plugins-android/pull/50

ETA: end of this week.
